src/components/create-safe/status/hooks/useWatchSafeCreation.ts

```diff
@@ -4,24 +4,27 @@ import { useRouter } from 'next/router'
 import { pollSafeInfo } from '@/components/create-safe/status/usePendingSafeCreation'
 import { AppRoutes } from '@/config/routes'
 import { SafeCreationStatus } from '@/components/create-safe/status/useSafeCreation'
-import useChainId from '@/hooks/useChainId'
 import { trackEvent, CREATE_SAFE_EVENTS } from '@/services/analytics'
+import { useAppSelector } from '@/store'
+import { selectChainById } from '@/store/chainsSlice'
 
 const useWatchSafeCreation = ({
   status,
   safeAddress,
   pendingSafe,
   setPendingSafe,
   setStatus,
+  chainId,
 }: {
   status: SafeCreationStatus
   safeAddress: string | undefined
   pendingSafe: PendingSafeData | undefined
   setPendingSafe: Dispatch<SetStateAction<PendingSafeData | undefined>>
   setStatus: Dispatch<SetStateAction<SafeCreationStatus>>
+  chainId: string
```

Comment:
It's possible to use `useChainId` instead of passing it.

Reply:
> It's possible to use `useChainId` instead of passing it.

Why? The place where the hook is used already has chainId in scope. Plus, with an explicit parameter, it's easier to test than having to mock internals

Reply:
I didn't realise it's just used the once.